polar-wasm-api/tests/polar.rs

```diff
@@ -93,10 +93,10 @@ fn new_query_from_str_succeeds() {
 #[wasm_bindgen_test]
 fn new_query_from_str_errors() {
     let polar = polar_wasm_api::Polar::wasm_new();
-    let mut query = polar.wasm_new_query_from_str("x").unwrap();
+    let mut query = polar.wasm_new_query_from_str("[]").unwrap();
     let err: Error = query.wasm_next_event().unwrap_err().dyn_into().unwrap();
     assert_eq!(err.name(), "RuntimeError::TypeError");
-    assert_eq!(err.message(), "trace (most recent evaluation last):\n  in query at line 1, column 1\n    x\nType error: can't query for: x at line 1, column 1");
+    assert_eq!(err.message(), "trace (most recent evaluation last):\n  in query at line 1, column 1\n    []\nType error: [] isn\'t something that is true or false, so can\'t be a condition at line 1, column 1");
```

Comment:
doesn't have to be in this PR, but we should fix the comma splice in this error message